packages/server/lib/controllers/reporter.js

```diff
@@ -1,14 +0,0 @@
-const send = require('send')
-/**
- * @type {import('@packages/resolve-dist')}
- */
-const { resolveDistPath } = require('@packages/resolve-dist')
```

Comment:
In converting this file to TypeScript, it surfaced that this resolveDistPath method doesn’t exist which led me to further uncover that this file is clearly no longer used!